NEWS.md

```diff
@@ -21,7 +21,12 @@ OpenSSL 3.1
 
 ### Major changes between OpenSSL 3.1.1 and OpenSSL 3.1.2 [under development]
 
-  * none
+  * By default, the FIPS provider is installed to ensure compliance in
```

Comment:
to ensure FIPS-140-3 requirements as date of ...?

Reply:
or just

> to ensure FIPS-140-3 requirements

without a date?

Reply:
As the requirements are a moving target, I think we should be somehow specific.

Reply:
Both of these are in effect already.  Instead of a date, I'll add a reference to the relevant IG for each.